Cody-Durham/ideabox-boilerplate
# Features/basic architecture

### What does this PR do?
- Adds functionality to save card button:
  -Idea data model is updated, cards appear on page, input fields clear
### What does it fix?
- NA
### Is this a feature or a fix?
- Feature
### Where should the reviewer start?
- Main js line 1-36
- Idea.js line 1-21
### How should this be tested?
- Fill out fields and press 'save' button. Idea cards should appear and input fields clear. Repeat multiple times to see cards fill up "card grid"
